### PR TITLE
Version checking

### PR DESCRIPTION
**Pull request by alexburlton-sonocent**
Initial stab at some kind of version check, so daktari doesn't explode with some meaningless python exception when you try to run an old version on a project.

It's a *little* hacky - I've used a regex to parse the version out of the config file because trying to interpret it as python is not necessarily safe if the daktari version is too old. An alternative I considered but rejected was having a separate config file to specify the version - that seemed more sucky from a user perspective (who wants multiple config files?). 

Open to other suggestions :grin:

### CODE REVIEW

File: daktari/config.py [hunk 1]
```diff
@@ -1,19 +1,69 @@
...
+
+    if not isinstance(required_version, version.Version):
+        print(red(f"❌  Invalid daktari_version in {config_path}: {required_version}"))
+        return False
+
+    logging.debug(f"Doing version check. Mine [{my_version}], required [{required_version}]")
+    if required_version > my_version:
+        print(
+            red(
+                f"""❌  Installed version of daktari [{my_version}] is too old
```
Comment (mattrussell-sonocent) [inline daktari/config.py:62]: Felt a bit odd to line break here to me - maybe a one liner is fine?

**Comment by mattrussell-sonocent** [comment]
> It's a little hacky - I've used a regex to parse the version out of the config file because trying to interpret it as python is not necessarily safe if the daktari version is too old. An alternative I considered but rejected was having a separate config file to specify the version - that seemed more sucky from a user perspective (who wants multiple config files?).

Yeah, the regex is an entirely reasonable fudge IMO.

Could we update the README to include `daktari_version = ...` as part of the example config (and perhaps it can be part of the list of files that `bump-version` manages?)

**Review by mattrussell-sonocent — APPROVED** [review]
LGTM

File: daktari/config.py (hunk 1 above)
Comment (alexburlton-sonocent) [reply daktari/config.py]: The `black` git hook did this, I've tried fighting it a bit but seems pretty keen on this indentation ._.

**Pull request merged**